src/layout/components/page-top.js

```diff
@@ -49,8 +50,10 @@ export class PageTop extends React.Component {
   renderSearch() {
     return (
       <div className="search">
-        <i className="ion-ios-search-strong"></i>
-        <input id="searchInput" type="text" placeholder="Search for..." />
+        <SearchBar>
```

Comment:
It looks like SearchBar doesn't do anything with `this.props.children` in that component, so there shouldn't be anything inside the SearchBar in this component